gentleman/src/pages/homepage/Homepage.jsx

```diff
@@ -0,0 +1,84 @@
+import "./homepage.css";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import { useEffect, useState } from "react";
+const Homepage=()=>{
+  const [category,setCategory]=useState([]);
```

Comment:
Suggestion - Instead of created 2 state variables, you can create a state and object and store both.